model/vm_host.rb

```diff
@@ -23,6 +23,11 @@ class VmHost < Sequel::Model
   include HealthMonitorMethods
   semaphore :checkup, :reboot, :destroy
 
+  VM_HOST_NVME_DISK_COMMAND = "nvme list"
+  # Google's public DNS
+  NSLOOKUP_IPV4_COMMAND = "nslookup 8.8.8.8"
+  NSLOOKUP_IPV6_COMMAND = "nslookup -type=AAAA 2001:4860:4860::8888"
```

Comment:
instead of nslookup, let's ping a hostname and make sure the ping goes through. This way, the dns resolution is already handled anyway.